config/routes.rb

```diff
@@ -3,4 +3,17 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+  get "/chef/:id", to: "chef#show"
+  get "/chef/:id/ingredients", to: "ingredients#index"
+  post "/chef/:id/dishes/:id", to: "dish_ingredient#destroy"
+  delete "/chef/:id/dishes/:id/ingredients/:id", to: "dish_ingredients#destroy", as: :remove
+
+  resources :chef, only: [:show, :create] do
+    resources :dishes, only: [:show, :create, :update]
+  end
+  
+  resources :chef, only: [:show]
+
+  
+
```

Comment:
There's a lot going on in this file. You have several duplicate routes going to the same place, either hand-rolled or using the `resources` syntax. While mixing and matching between hand-rolled and non-hand-rolled routes is fine I would say to make sure you remove the dupes.